tests/cases/conformance/es2020/modules/exportAsNamespace5.ts

```diff
@@ -0,0 +1,12 @@
+// @module: esnext
+// @moduleResolution: bundler
+
+// @filename: three.d.ts
+export type Named = 0;
```

Comment:
Please could we also test the value land namespace properties. Either in this test case or as a separate one.

https://www.typescriptlang.org/dev/bug-workbench/?moduleResolution=99&module=199&emitDeclarationOnly=true&ts=5.2.2#code/PTAEAEDMEsBsFMB2BDAtvAXKALgCwE7zwB0AJsdgM4CwAUKfAMazKGiMD2il2oAcmniksABgDcdeAA8ADh3y8A3v0GlQAXwm06ICDAQp0WbAHcOZCjVrS5C0ACpQySqAaRkAV1i9I+DqlAAImJgPEJ4QK0dMCg4JEEsLhIqOmhUW15uUF9-IJDTDki6bmIBdFIxIA